addons/website_sale/views/templates.xml

```diff
@@ -1557,32 +1557,34 @@
                              name="website_sale_cart_line_quantity">
                             <t t-if="not line._is_not_sellable_line()">
                                 <t t-if="show_qty">
-                                    <a href="#"
-                                       class="js_add_cart_json btn btn-link d-inline-block border-end-0"
-                                       aria-label="Remove one"
-                                       title="Remove one">
-                                        <i class="position-relative z-index-1 fa fa-minus"/>
-                                    </a>
-                                    <input type="text"
-                                           class="js_quantity quantity form-control border-start-0 border-end-0"
-                                           t-att-data-line-id="line.id"
-                                           t-att-data-product-id="line.product_id.id"
-                                           t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty"/>
-                                    <t t-if="line._get_shop_warning(clear=False)">
-                                        <a href="#" class="btn btn-link">
-                                        <i class='fa fa-warning text-warning'
-                                           t-att-title="line._get_shop_warning()"
-                                           role="img"
-                                           aria-label="Warning"/>
+                                    <t class="o_wsale_adjust_quantity">
```

Comment:
My xpath was a pseudo xpath, you have to update it a bit to something like `expr="//t[@t-if='show_qty']"`

Reply:
Ahrgg my bad, I thought it was a feature of xpath xD

Reply:
@Feyensv I think I'm still wake up because I really don't see why it crash ^^
The xpath rule seems like the other in the codebase...
`Element '<xpath expr="//t[@t-if=&#34;show_qty&#34;]">' cannot be located in parent view`

Reply:
My fault :man_facepalming:, the first xpath modifies the element, which makes the second fail.  We'll still add a simple `name` on the `t` attribute to use as xpath hook.

Reply:
ohh then we are both waking up ahah

I could also use the contain function ?
 like this : `contains(@t-if, 'show_qty')`

Could be an issue if we use `show_qty` somewhere else but that not the case for now, what do you think ?